src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCauseDisplayData.java

```diff
@@ -52,10 +51,14 @@ public FailureCauseDisplayData() {
     /**
      * Use this constructor when the build is known.
      *
-     * @param build corresponding build
+     * @param parentUrl url of the parent job.
+     * @param parentName name of the parent job.
+     * @param buildUrl url of the build.
+     * @param buildName name of the build.
+     *
      */
-    public FailureCauseDisplayData(final Run build) {
-        links = new Links(build);
+    public FailureCauseDisplayData(final String parentUrl, final String parentName, final String buildUrl, final String buildName) {
```

Comment:
Breaking old signature of a public non restricted class, the old constructor and public methods needs to be kept but deprecated for binary compatibility.